src/content/docs/query-your-data/explore-query-data/query-builder/introduction-new-data-explorer.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Introduction to the data explorer
+tags:
+  - Query your data
+  - Explore and query data
+  - Data explorer
+metaDescription: 'The New Relic data explorer lets you run queries of your data without using NRQL-format queries'
+redirects:
+  - /docs/query-your-data/explore-query-data/query-builder/introduction-query-builder/#query-from-anywhere
+
+freshnessValidatedDate: never
+---
+
+You can now explore all your data visually using the <DNT>**query experience data explorer**</DNT> without needing to know how to write [NRQL-format queries](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
+
+Need to perform a more detailed search? Read how to [query your data](/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data/). If you haven't already, create your free New Relic account to start monitoring your data today.
+
+
+## Why it matters [#why-it-matters]
+
+Do you research the state of your systems? Do you need to plan resources, identify and respond to incidents, or troubleshoot faulty behaviors? The data explorer makes it easy to identify, fetch and visualize the data you are looking for through visual menus, without ever using NRQL or building queries.
+
+With the <DNT>**data explorer**</DNT>, you can access all data stored in the New Relic database (NRDB) in a quick, intuitive way. You can then select facet your queries using the attributes and filter down to the needed value.
+
+Other things the data explorer view helps you:
+
+* View data from different perspectives: from raw data to various visualizations that provide insights on evolution, distribution, and more.
+* Drill down into data using filters.
+* Add your searches to a dashboard.
+* Understand how NRQL works: the Data Explorer demonstrates how queries are constructed.
+
+### Types of data you can explore [#types-of-data]
+
+Following are the types of data you can explore using the <DNT>**Data Explorer**</DNT> view:
+
+* [Dimensional metrics](/docs/data-apis/understand-data/new-relic-data-types/#dimensional-metrics) (the Metric data type).
+* [Events](/docs/data-apis/understand-data/new-relic-data-types/#events-new-relic) (the Event data type).
+* [Logs](/docs/logs/ui-data/use-logs-ui) (the Log data type).
+
+## Exploring your data [#exploring-your-data]
```

Comment:
We don't use -ing formations for h2s. Please change to active voice, eg "Explore your data"